tooling/reproducible/repro_process.sh

```diff
@@ -23,7 +23,7 @@ OS="$2"
 
 expandJDK "$JDK_DIR" "$OS"
 
-removeGeneratedClasses "$JDK_DIR" "$OS"
+removeExcludedFiles "$JDK_DIR"
```

Comment:
this was called GeneratedClasses, as it related to just cleaning the generated CDS artifacts